### DISCUSSION TITLE
Best practice to add a new Client Handler

### DISCUSSION THREAD

**Discussion by vramULB** [cmu-sei/GHOSTS#354] [General]
Hello again :)

I'm working for the Ghost Linux Client, and in my configuration I've implemented a Mattermost Browser service and I would like to link this service to a LLM to do actions(or at least adapt the agent timelines), I already ask about it in another conversation.
The point is that, I'm can create an agent of my own that links Mattermost and LLM without going through the Ghosts API, but I'm trying to have a community perspective so that maybe in the future you can add a solution to your project. 
Which could be the best way to implement a new Client Handler ?

**Comment by sei-dupdyke** [comment]
GHOSTS SHADOWS coming very soon to do this!
